.github/pull_request_template.md

```diff
@@ -15,6 +15,7 @@ Please click the `Preview` tab and select a different PR template if your change
 - [ ] I have reviewed visual test updates properly before approving.
 - [ ] If changes will affect consumers of the package, I have created a changeset entry.
 - [ ] If a changeset file has been created, I have tested these changes in [PIE Aperture](https://github.com/justeattakeaway/pie-aperture/) using the `/test-aperture` command.
+- [ ] I have filled out the DS Review Tracker checklist (Moving PR to "Ready to Review")
```

Comment:
Is it worth including a link to the PR list?

Reply:
It's on every PR on the right-hand side (under "Projects"), so should be able to edit it in the PR directly  😄